public/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
```

Comment:
I assume this downloads all of their icons? If you look at their docs here it shows how to use fa in react and it only bundle the specific icons we end up using:
https://fontawesome.com/how-to-use/on-the-web/using-with/react

Reply:
In terms of understanding why this matters, the the [performance](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance) tab in dev tools can show you the performance impact of this, here you can see the text loads with no icon, then a broken icon, before loading:
![perf mov](https://user-images.githubusercontent.com/9036408/95693460-681cb580-0c24-11eb-9ea3-7df79e1ce482.gif)

By importing just the icon we need the icon loads immediately: 
![more-perf mov](https://user-images.githubusercontent.com/9036408/95693727-bc746500-0c25-11eb-92d7-ce87a13dd728.gif)

Reply:
This is a very good point, and the doc looks straightforward enough. I'll most likely give this a go and make it a separate pull request

Reply:
#8 I have created a separate pull request for this update here.

Reply:
Nice, do you want to merge that PR into this one or merge this first?

Reply:
Probably merge #8 into this one first. There seem to be some conflicts on this PR at the moment, I'm guessing that's due to the directory rearrange?